### DISCUSSION TITLE
Advanced Custom Fields ==> Secure Custom Fields

### DISCUSSION THREAD

**Discussion by winnewoerp** [flyntwp/flynt#582] [General]
Being new to flynt I asked myself if
1) I really need to have ACF Pro available inside my dev environment and
2) if WP's announcment of its own ACF fork called "Secure Custom Fields" https://wordpress.org/news/2024/10/secure-custom-fields/ will have an impact on future flynt developments.

What are your thoughts about this?

Well, thanks for flynt in any case.

Best wishes from Hamburg

**Comment by timohubois** [comment]
@winnewoerp I think technically it should work – have not tested it yet due for me its more an ethical question and I'm happy with ACF Pro.

**Comment by winnewoerp** [comment]
Thx! Does that mean that SCF should work technically (as of today) but without the exclusive ACF Pro stuff (e.g. repeater fields) that flynt partly relies on (if I got it right)?

**Comment by steffenbew** [comment]
Hey @winnewoerp, welcome to Flynt and thanks for asking! 

I'm happy to answer your questions:

1. Yes, you need to have ACF Pro installed. Flynt relies on the repeater field and other functionalities that are part of the Pro version.

2. The announcement to fork ACF doesn't have an impact on future Flynt developments.

All the best!

**Reply by winnewoerp** [reply]
OK, that's good to know, thank you.
